requirements.txt

```diff
@@ -2,3 +2,4 @@ InstagramApi
 python-twitter
 requests
 facebook-sdk
+configparser
```

Comment:
I don't think `configparser` has to be added here, it is in the python standard library